src/uct/ib/rc/base/rc_iface.c

```diff
@@ -199,6 +214,21 @@ UCS_CLASS_INIT_FUNC(uct_rc_iface_t, uct_ib_iface_ops_t *ops, uct_pd_h pd,
     ucs_arbiter_init(&self->tx.arbiter);
     ucs_list_head_init(&self->ep_list);
 
+    /* Check FC parameters correctness */
+    if ((config->fc.sreq_factor <=0) || (config->fc.sreq_factor >= 1)) {
+        ucs_error("The factor for soft FC credit request must be: (0, 1)");
```

Comment:
* "must be in the range (0..1)" - not sure all users are familiar with the math notation..
* pls add space after <=